### DISCUSSION TITLE
Password must be asked when I launch github application on mobile

### DISCUSSION THREAD

**Discussion by basarballioz** [community/community#61723] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.122.0 (104639223)
Device: iPhone15,2 (iOS 16.5.1)
</details>

**Comment by aconsuegra** [comment]
Hi @basarballioz, thanks for reaching out with your feature request! I will pass it to the team, so we can evaluate it. Keep the feedback coming!

**Reply by basarballioz** [reply]
Thanks a lot, I am looking forward !
